plugins/policy-recommendation/policy_recommendation_job.py

```diff
@@ -811,13 +824,22 @@ def write_recommendation_result(
         recommendation_id = str(uuid.uuid4())
     else:
         recommendation_id = recommendation_id_input
-    result_dict = {
-        "id": recommendation_id,
-        "type": recommendation_type,
-        "timeCreated": datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
-        "yamls": "---\n".join(filter(None, result)),
-    }
-    result_df = spark.createDataFrame([result_dict])
+
+    result_dict_list = []
+    for key, value in result.items():
+        for item in value:
+            if item is not None:
```

Comment:
minor python nit: unless you explicitly need to compare with None, you can just use "if item:"

Reply:
Updated, thanks!